app/views/journeys/abroad/SelfEmploymentAbroadView.scala.html

```diff
@@ -14,25 +14,25 @@
  * limitations under the License.
  *@
 
-@import controllers.actions.AuthenticatedIdentifierAction.User
-@import controllers.journeys.abroad.routes.SelfEmploymentAbroadController
+@import controllers.journeys.abroad.routes
 @import models.common.{BusinessId, TaxYear, UserType}
+@import views.html.components.SubmitButton
 @import views.html.templates.helpers.Heading
 
 @this(
-    layout: templates.Layout,
-    formHelper: FormWithCSRF,
-    govukErrorSummary: GovukErrorSummary,
-    govukRadios: GovukRadios,
-    govukButton: GovukButton,
-    heading: Heading
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukRadios: GovukRadios,
+        submitButton: SubmitButton,
+        heading: Heading
 )
 
 @(form: Form[_], taxYear: TaxYear, businessId: BusinessId, userType: UserType, mode: Mode)(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = title(form, messages(s"selfEmploymentAbroad.title.$userType"))) {
 
-    @formHelper(action = SelfEmploymentAbroadController.onSubmit(taxYear, businessId, mode), Symbol("autoComplete") -> "off") {
+    @formHelper(action = routes.SelfEmploymentAbroadController.onSubmit(taxYear, businessId, mode), Symbol("autoComplete") -> "off") {
 
```

Comment:
importing the routes in this way stops errors when using IntelliJ Ultimate edition